pkg/controller/statefulset/stateful_set_control_test.go

```diff
@@ -136,6 +138,9 @@ func runTestOverPVCRetentionPolicies(t *testing.T, testName string, testFn func(
 }
 
 func pvcDeletePolicyString(policy *apps.StatefulSetPersistentVolumeClaimRetentionPolicy) string {
+	if policy == nil {
+		return "Retain"
```

Comment:
We use this string to describe tests only, so let's use something like "nullPolicy" instead so it's not confused with other test cases.

Reply:
Updated.